chimney-macro-commons/src/main/scala-3/io/scalaland/chimney/internal/compiletime/ExprsPlatform.scala

```diff
@@ -160,11 +160,15 @@ private[compiletime] trait ExprsPlatform extends Exprs { this: DefinitionsPlatfo
     def suppressUnused[A: Type](expr: Expr[A]): Expr[Unit] = '{ val _ = ${ expr } }
 
     def prettyPrint[A](expr: Expr[A]): String =
-      expr.asTerm
-        .show(using Printer.TreeAnsiCode)
-        // remove $macro$n from freshterms to make it easier to test and read
-        .replaceAll("\\$macro", "")
-        .replaceAll("\\$\\d+", "")
+      try
+        expr.asTerm
+          .show(using Printer.TreeAnsiCode)
+          // remove $macro$n from freshterms to make it easier to test and read
+          .replaceAll("\\$macro", "")
+          .replaceAll("\\$\\d+", "")
+      catch {
+        case e: StackOverflowError => expr.toString
+      }
```

Comment:
For some reason, when working with cases that this PR attempts to fix, I sometimes get StackOverflow from `expr.asTerm.show(using Printer.TreeAnsiCode)` :O